tests/unit/views/notifications/test_organization_admin_notification.py

```diff
@@ -0,0 +1,226 @@
+"""Test template strings in project crew membership notifications."""
+# pylint: disable=too-many-arguments
+
+from pytest_bdd import given, parsers, scenarios, then, when
+
+from funnel import models
+from funnel.models.membership_mixin import MEMBERSHIP_RECORD_TYPE
+
+scenarios('organization_admin_notification.feature')
+
+
+@given(
+    "Vetinari is an owner of the Ankh-Morpork organization",
+)
+def given_vetinari_owner_org(user_vetinari, org_ankhmorpork) -> None:
+    assert 'owner' in org_ankhmorpork.roles_for(user_vetinari)
+
+
+@given("Vimes is an admin of the Ankh-Morpork organization")
+def given_vimes_admin(db_session, user_vimes, org_ankhmorpork, user_vetinari) -> None:
+    vimes_admin = models.OrganizationMembership(
+        user=user_vimes,
+        organization=org_ankhmorpork,
+        granted_by=user_vetinari,
+        is_owner=False,
+    )
+    db_session.add(vimes_admin)
+    assert 'admin' in org_ankhmorpork.roles_for(user_vimes)
+
+
+@when(
+    parsers.parse("Vetinari adds Ridcully with the role {role}"),
+    target_fixture='ridcully_admin',
+)
+@given(
+    parsers.parse(
+        "Ridcully is an existing admin with roles {role} of the Ankh-Morpork organization"
+    ),
+    target_fixture='ridcully_admin',
+)
+def when_vetinari_adds_ridcully(
+    db_session, user_vetinari, user_ridcully, org_ankhmorpork, role
+):
+    is_owner = 'owner' in role
+    ridcully_admin = models.OrganizationMembership(
+        user=user_ridcully,
+        organization=org_ankhmorpork,
+        granted_by=user_vetinari,
+        is_owner=is_owner,
+    )
+    db_session.add(ridcully_admin)
+    db_session.commit()
+    return ridcully_admin
+
+
+@then(
+    parsers.parse("{user} gets notified with {notification_string} about the addition")
+)
+@then(
+    parsers.parse(
+        "{user} gets notified with {notification_string} about the invitation"
+    )
+)
+@then(
+    parsers.parse(
+        "{user} gets notified with {notification_string} about the acceptance"
+    )
+)
+@then(parsers.parse("{user} gets notified with {notification_string} about the change"))
+def then_user_gets_notification(
+    user,
+    notification_string,
+    user_vimes,
+    user_ridcully,
+    user_vetinari,
+    ridcully_admin,
+) -> None:
+    user_dict = {
+        "Ridcully": user_ridcully,
+        "Vimes": user_vimes,
+        "Vetinari": user_vetinari,
+    }
```

Comment:
Replace the user fixtures with `<user>_admin` fixtures and use `fixture.user` in the dict.